src/Text/Markdown/SlamDown/Parser/Inline.purs

```diff
@@ -285,10 +285,10 @@ inlines = many inline2 <* eof
           pure { date, time }
 
         parseDateValue = do
-          month <- natural
-          skipSpaces *> dash *> skipSpaces
           day <- natural
           skipSpaces *> dash *> skipSpaces
+          month <- natural
+          skipSpaces *> dash *> skipSpaces
```

Comment:
depending on who's coding this, it might look right or wrong! (US vs UK). But it's just names, and no logic depends on the specific identity; all that matters is that we are consistent within the library.